pywbem/config.py

```diff
@@ -60,3 +61,11 @@
 #: operations.
 
 DEFAULT_ITER_MAXOBJECTCOUNT = 1000
+
+#: Maximum log entry size. An integer that sets the maximum size of each log
+#: entry if the log_detail_level flag is set.
+
+DEFAULT_MAX_LOG_ENTRY_SIZE = 1000
```

Comment:
First, moved it to _logging.py.  Second, right now we have no way to allow a user to c hange this so it is really the max so I changed the name.